slices/fonts-dejavu-core.yaml

```diff
@@ -13,7 +15,6 @@ slices:
 
   config:
     essential:
-      - fonts-dejavu-core_fonts
```

Comment:
even if this is the right choice, can we please keep it? Otherwise this might introduce a regression for those already using this slice.